### PR TITLE
fix spelling error

### PR DESCRIPTION
**Pull request by mahanstreamer**
;)

### CODE REVIEW

**Comment by tmatth** [comment]
Cool, feel free to fix these `cargo-spellcheck` results as well:
```
error: spellcheck(Hunspell)
    --> /big-repos/rust-av/bitstream/src/bitread.rs:130
     |
 130 |  Alignes the bits present in the internal buffer.
     |  ^^^^^^^
     | - Aligned, Aligns, Aligners, Aligner, Align es, Align-es, Lignaloes, or one of 3 others
     |
     |   Possible spelling mistake found.

error: spellcheck(Hunspell)
   --> /big-repos/rust-av/codec/src/error.rs:25
    |
 25 |  A specialised `Result` type for coding operations.
    |    ^^^^^^^^^^^
    | - specialized, specialistic, specialism, specialist, or specialize
    |
    |   Possible spelling mistake found.

error: spellcheck(Hunspell)
   --> /big-repos/rust-av/data/src/audiosample.rs:24
    |
 24 |  Alignes a value to a specific number of bytes.
    |  ^^^^^^^
    | - Aligned, Aligns, Aligners, Aligner, Align es, Align-es, Lignaloes, or one of 3 others
    |
    |   Possible spelling mistake found.

error: spellcheck(Hunspell)
    --> /big-repos/rust-av/data/src/frame.rs:560
     |
 560 |  A specialised type for reference-counted `Frame`
     |    ^^^^^^^^^^^
     | - specialized, specialistic, specialism, specialist, or specialize
     |
     |   Possible spelling mistake found.

error: spellcheck(Hunspell)
    --> /big-repos/rust-av/data/src/pixel.rs:205
     |
 205 |  All YUV color reprentations.
     |                ^^^^^^^^^^^^^
     | - representations, representation, presentations, or presentation
     |
     |   Possible spelling mistake found.

error: spellcheck(Hunspell)
    --> /big-repos/rust-av/data/src/pixel.rs:285
     |
 285 |  Vertial subsampling in power of two
     |  ^^^^^^^
     | - Tertial, Vertical, or Inertial
     |
     |   Possible spelling mistake found.

error: spellcheck(Hunspell)
   --> /big-repos/rust-av/format/src/error.rs:19
    |
 19 |  A specialised `Result` type for muxing/demuxing operations.
    |    ^^^^^^^^^^^
    | - specialized, specialistic, specialism, specialist, or specialize
    |
    |   Possible spelling mistake found.

```

**Comment by mahanstreamer** [comment]
done ;)

**Review by Luni-4 — APPROVED** [review]
It seems good to me! Thank you for your contribution! :)

Let's wait for a @tmatth comment before merging it though

**Comment by mahanstreamer** [comment]
I guess the subsampling change ended up here as well. I guess you could also consider that a typo ;)

File: data/src/pixel.rs [hunk 1]
```diff
@@ -375,12 +375,12 @@ impl Chromaton {
...
     pub fn get_width(self, width: usize) -> usize {
-        (width + ((1 << self.v_ss) - 1)) >> self.v_ss
+        (width + ((1 << self.h_ss) - 1)) >> self.h_ss
     }
 
     /// Calculates the height for a component from general image height.
     pub fn get_height(self, height: usize) -> usize {
-        (height + ((1 << self.h_ss) - 1)) >> self.h_ss
+        (height + ((1 << self.v_ss) - 1)) >> self.v_ss
     }
```
Comment (tmatth) [inline data/src/pixel.rs:384]: This commit should be in a separate PR since it's not just a spelling fix.

Comment (mahanstreamer) [reply data/src/pixel.rs:384]: fine

**Comment by Luni-4** [comment]
@mahanstreamer 

Please rebase this PR and then I suppose we are ready to land it

**Comment by Luni-4** [comment]
@mahanstreamer 

Your PR has been merged, thanks for your work and thanks to @tmatth for the review! :)
